core/common/src/MapImagerySettings.ts

```diff
@@ -6,47 +6,221 @@
  * @module DisplayStyles
  */
 
-import { BackgroundMapProps, BackgroundMapSettings } from "./BackgroundMapSettings";
+import { BackgroundMapProviderName, BackgroundMapType } from "./BackgroundMapSettings";
 import { ColorDef, ColorDefProps } from "./ColorDef";
 import { MapLayerProps, MapLayerSettings } from "./MapLayerSettings";
 
-/** The JSON representation of base layer properties -- these can be represented by either a full map layer or a simple color.
- * @beta
- */
-export type BaseLayerProps = MapLayerProps | ColorDefProps;
-
 /** The JSON representation of the map imagery.  Map imagery include the specification for the base layer (which was originally
  * represented by [[BackgroundMapProps.providerName]]  && [[BackgroundMapProps.providerData]]) and additional map layers.
  * In earlier versions only a background map was supported as specified by the providerName and mapType members of [[BackgroundMapSettings]] object.
  * In order to provide backward compatibility the original [[BackgroundMapSettings]] are synchronized with the [[MapImagerySettings]] base layer as long as
  * the settings are compatible.
  * @beta
  */
-export interface MapImageryProps {
-  backgroundBase?: BaseLayerProps;
-  backgroundLayers?: MapLayerProps[];
-  overlayLayers?: MapLayerProps[];
+
+export interface BackgroundMapProviderProps {
```

Comment:
Objects of this type will never be saved into the iModel or into a saved view, correct?

Reply:
Yes it should be saved.